hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ApiPermissionConfig.java

```diff
@@ -0,0 +1,320 @@
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange createTopic,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange updateTopic,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange deleteTopic,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange submitMessage,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange getTopicInfo,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange ethereumTransaction,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange scheduleCreate,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange scheduleSign,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange scheduleDelete,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange scheduleGetInfo,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenCreate,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenFreezeAccount,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenUnfreezeAccount,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenGrantKycToAccount,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenRevokeKycFromAccount,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenDelete,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenMint,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenBurn,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenAccountWipe,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenUpdate,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenGetInfo,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenAssociateToAccount,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenDissociateFromAccount,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenGetNftInfo,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenGetNftInfos,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenGetAccountNftInfos,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenFeeScheduleUpdate,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenPause,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange tokenUnpause,
+        @ConfigProperty(defaultValue = "0-*") PermissionedAccountsRange getVersionInfo,
+        @ConfigProperty(defaultValue = "2-50") PermissionedAccountsRange networkGetExecutionTime,
+        @ConfigProperty(defaultValue = "2-59") PermissionedAccountsRange systemDelete,
+        @ConfigProperty(defaultValue = "2-60") PermissionedAccountsRange systemUndelete,
```

Comment:
Is this correct?  From how I read the documentation, it's three specific accounts (2,50,60), rather than a range.
Some of the other items here are similar.

Reply:
I copied that values from the `api-permission.properties` file that is located in the mono-service: https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/resources/api-permission.properties

Reply:
Might be the documentation is out of date, or the implementation is a little overly permissive.  Something to look into in future work.